web/Views/Admin/Index.cshtml

```diff
@@ -19,11 +29,17 @@
             else
             {
                 <p>You are not an admin :(</p>
+                @using (Html.BeginForm("RequestAdminAccess", "Admin", FormMethod.Post))

```

Comment:
Could you add a screenshot of what this change looks like? Good practice to do this for any frontend changes